lib/transaction.js

```diff
@@ -28,17 +28,19 @@ function mixinTransaction(DB2, db2) {
       });
     }
 
+    self.connStr += ';IsolationLevel=ReadCommitted';
+
     self.client.open(self.connStr, function(err, connection) {
       if (err) return cb(err);
       connection.beginTransaction(function(err) {
-        if (isolationLevel) {
-          var sql = 'SET CURRENT ISOLATION ' + isolationLevel;
-          connection.query(sql, function(err) {
-            cb(err, connection);
-          });
-        } else {
-          cb(err, connection);
-        }
+        // if (isolationLevel) {
+        //   var sql = 'CHANGE ISOLATION TO CS'; // + isolationLevel;
+        //   connection.query(sql, function(err) {
+        //     cb(err, connection);
+        //   });
+        // } else {
+        cb(err, connection);
+        // }
```

Comment:
The code to control isolation is commented out. Is it intentional? If yes, the commented code should be removed and we should add a comment to explain how isolation is set.